app/src/main/java/com/babylon/wallet/android/MainActivity.kt

```diff
@@ -46,7 +51,18 @@ class MainActivity : FragmentActivity() {
                     derivedStateOf { DevBannerState(isVisible = isDevBannerVisible) }
                 }
 
-                DevelopmentPreviewWrapper(devBannerState = devBannerState) { padding ->
+                var linkConnectionsStatus: LinkConnectionsStatus? = null
+                if (BuildConfig.EXPERIMENTAL_FEATURES_ENABLED) {
+                    val isLinkConnectionsStatusEnabled by linkConnectionStatusObserver.isEnabled.collectAsState()
```

Comment:
Probably not an issue or doesn't make any difference but in some places we use `collectAsStateWithLifecycle` and sometimes  `collectAsState` whereas as far as I remember we should use `collectAsStateWithLifecycle` ? 
Correct me if Im wrong